src/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Not sure if this is needed.

Reply:
Most of the guides start with the statement that two lines are required: version and project name. And if I remove it GUI generator starts giving me warnings. Just not sure what version is required for current version of the file. Think the only problem was that in Ububtu cmake version was lower. I'll check it in a bit.

Reply:
This is required for main `CMakeLists.txt` but I am not sure about this one in subdirectory.